migrat.md

```diff
@@ -0,0 +1,39 @@
+### Migration
+by Jack
+2016年 12月 14日 星期三 11:14:48 CST
```

Comment:
Suggest to use ISO 8601 format as here are the only Chinese characters.

[1] https://en.wikipedia.org/wiki/ISO_8601